lib/cisco_node_utils/cmd_ref/bgp.yaml

```diff
@@ -191,12 +191,13 @@ graceful_restart_timers_stalepath_time:
 
 log_neighbor_changes:
   kind: boolean
-  #TODO: Equivalent for bgp is 'bgp log'
-  # Currently supported but broken in IOS XR
+  default_value: false
+  cli_ios_xr:
+    config_get_token_append: '/^bgp log neighbor changes disable$/'
+    config_set_append: ' <state> bgp log neighbor changes disable'
```

Comment:
Mistaken assumption that true equates to enabling the disable keyword. Reworking - and will take a look as enforce-first-as just to check.